pkg/extensions/extension_search.go

```diff
@@ -103,7 +103,7 @@ func GetExtensions(config *config.Config) distext.ExtensionList {
 	if config.Extensions != nil && config.Extensions.Search != nil {
 		endpoints := []string{constants.ExtSearchPrefix}
 		searchExt := getExtension("_zot",
-			"https://github.com/project-zot/zot/tree/main/pkg/extensions/_zot.md",
+			"https://github.com/project-zot/zot/blob/"+config.ReleaseTag+"/pkg/extensions/_zot.md",
 			"zot registry extension",
```

Comment:
Unrelated to your PR, but also make this "zot registry extensions" <- plural

Reply:
Added as requested